packages/loader/container-loader/src/container.ts

```diff
@@ -442,10 +442,6 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         this._deltaManager.connectionManager.forceReadonly(readonly);
     }
 
-    public get id(): string {
```

Comment:
@alpaix, did you figure out plan for FRS / 3p flows RE usage of doc IDs? Want to make sure that this PR does not come as a surprise to you.